### ISSUE TITLE
Azure RHEL: service loadBalancer not accessible from outside of the cluster

### ISSUE THREAD

**Issue by vgramer** [kubermatic/kubermatic#8768]
In a KKP user cluster with one worker node and with CIlium CNI or Canal CNI + IPVS kube-proxy mode.
- create a load balancer service with a backend pod
- from the local computer try connecting to external-ip of the load balancer
- connection is refused

The service
```
 $ k -n whoami get svc whoami-lb                                                                                                                          
NAME        TYPE           CLUSTER-IP      EXTERNAL-IP    PORT(S)        AGE
whoami-lb   LoadBalancer   10.240.29.147   20.71.174.73   80:32419/TCP   128m
````
try connecting from local computer
```
$ curl 20.71.174.73
curl: (7) Failed to connect to 20.71.174.73 port 80: Connection refused
```

if we trace the packet on the worker node, we can see the request is dropped.
```
tcpdump -nni any host <LocalComputerIP> and port not 22
dropped privs to tcpdump
tcpdump: verbose output suppressed, use -v or -vv for full protocol decode
listening on any, link-type LINUX_SLL (Linux cooked v1), capture size 262144 bytes
10:49:34.057673 IP <LocalComputerIP> .50117 > 20.71.174.73.80: Flags [S], seq 606674401, win 65535, options [mss 1460,nop,wscale 6,nop,nop,TS val 1258695530 ecr 0,sackOK,eol], length 0
10:49:34.057795 IP 20.71.174.73 > <LocalComputerIP> : ICMP host 20.71.174.73 unreachable - admin prohibited filter, length 72
```

Firewalld is enabled:
```
firewall-cmd --list-all
public (active)
  target: default
  icmp-block-inversion: no
  interfaces: eth0
  sources:
  services: cockpit dhcpv6-client ssh
  ports: 8472/udp 30000-32767/tcp 30000-32767/udp
  protocols:
  masquerade: no
  forward-ports:
  source-ports:
  icmp-blocks:
  rich rules:
```

**Comment by embik** [comment]
I think the problem is that Azure's LBs have the "floating IPs" feature enabled (and I checked the CCM code; it's hardcoded), effectively sending packets with the LB IP address and port to the nodes. `firewalld` is blocking those requests, as they're not covered by any rule. I don't think we can craft a `firewall-cmd` command to run as part of machine bootstrapping, as the packet could have any public IP and any port.

The only way to fix this might be to disable `firewalld` on Azure.

**Comment by wurbanski** [comment]
Same thing happens with Cilium using iptables proxy mode.

**Comment by kron4eg** [comment]
Released as part of machine-controller v1.46.0 and integrated in Kubermatic since #9512 merged.
